src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,47 @@
+package duke.ui;
+
+import java.util.Scanner;
+
+/**
+ * Represents a class that deals with interactions with the user
+ */
+public class Ui {
+    public void showLoadingError() {
+        System.out.println("Something went wrong");
+    }
+
+    /**
+     * Prints the welcome message for the Duke application
+     */
+    public void showWelcome() {
+        // print welcome message
```

Comment:
Perhaps the comment here could be removed since this is a duplicate of the above!